trajoptlib/include/trajopt/DifferentialTrajectoryGenerator.hpp

```diff
@@ -97,22 +97,22 @@ class TRAJOPT_DLLEXPORT DifferentialTrajectorySample {
   /// The heading.
   double heading = 0.0;
 
-  /// The left wheel velocity.
+  /// The left wheels velocity.
```

Comment:
It was written that way because differential drives have only one wheel per side.

Reply:
:woman_facepalming: I meant to add a an apostrophe